packages/visualizations-react/stories/Table/options.ts

```diff
@@ -41,15 +55,15 @@ export const columns: Column[] = [
         key: 'isFeatured',
         dataFormat: 'boolean',
         options: {
-            display: (bool: boolean) => (bool ? 'αληθές' : 'ψευδές'),
+            valueToLabel: (v: boolean) => (v ? 'αληθές' : 'ψευδές'),
         },
     },
     {
         title: 'Word count',
         key: 'wordCount',
         dataFormat: 'number',
         options: {
-            display: (v: string) => `${v} words`,
+            valueToLabel: (v: number) => `${v} words`,
```

Comment:
```suggestion
            valueToLabel: (v: string) => `${v} words`,
```
I have a TS error: `Type 'string' is not assignable to type 'number'`